### PR TITLE
Fix/issue 452

### PR DESCRIPTION
**Pull request by jtlap**
Denis I have taken into account your remarks 
corrected the bug and made the index CS

### CODE REVIEW

File: test/unit/module/real/combinatorial/nth_prime/regular/nth_prime.hpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  TTS_EQUAL(eve::nth_prime(T(53)), T(251));
+
+  if constexpr( sizeof(elt_t) >= 2) TTS_EQUAL(eve::nth_prime(T(54)),  T(257));
+  else                              TTS_EQUAL(eve::nth_prime(T(54)),  T(0));
+
+  if constexpr( sizeof(elt_t) == 4)
+  {
+    using r_t = eve::detail::as_floating_point_t<T>;
+    TTS_EQUAL(eve::float32(eve::nth_prime)(T(1000)), r_t(7927));
+    TTS_EQUAL(eve::float32(eve::nth_prime)(T(9000)), r_t(93187));
```
Comment (DenisYaroshevskiy) [inline test/unit/module/real/combinatorial/nth_prime/regular/nth_prime.hpp:34]: Maybe `>=` 4?

**Comment by DenisYaroshevskiy** [comment]
LGTM.
I would ask @jfalcou to review - I don't have enough context for a more detailed review.

File: test/unit/module/real/combinatorial/nth_prime/regular/nth_prime.hpp (hunk 1 above)
Comment (jtlap) [reply test/unit/module/real/combinatorial/nth_prime/regular/nth_prime.hpp:34]: no problem

File: test/unit/module/real/combinatorial/nth_prime/regular/.#nth_prime.hpp [hunk 2]
```diff
@@ -0,0 +1 @@
+jt@jt-Inspiron-15-5501.76661
```
Comment (jfalcou) [inline test/unit/module/real/combinatorial/nth_prime/regular/.#nth_prime.hpp:1]: this is an useless file
Please remove

**Pull request merged**